content/docs/1.1.1/volumes-and-nodes/storage-tags.md

```diff
@@ -28,10 +28,18 @@ When creating a volume, specify the disk tag and node tag in the UI.
 
 ### Kubernetes
 
-Use Kubernetes StorageClass setting to specify tags.
+Use Kubernetes StorageClass setting to specify tags to mount volumes only on specified nodes.
 
-For example:
+The default setting could be enabled by editing [values.yaml](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/chart/values.yaml) `persistence.defaultNodeSelector` when install with Helm.
```

Comment:
1. NIT: The term/word `setting` may be confusing here. In Longhorn `setting` typically means Setting CRs.
2. It's better to re-arrange this sector a little bit. e.g.,
```
This feature can be enabled by setting one field for the Longhorn StorageClass:
One example here...
1. If you want to config this during Helm install, you can modify `values.yaml` or adding flag `......`
2. If you want to config this for the default Longhorn StorageClass, please add the fields by editing the ConfigMap `longhorn-storageclass` in namespace `longhorn-system`
3. Or you can create a customized StorageClass with the fields confined
......
```